src/components/grapher/PerSheetPreview.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div class="persheetpreview">
+    <svg class="persheetpreview-svg"><g>
```

Comment:
I think for now, we should park the sheet preview feature. I feel that `Grapher` is going to be changing a lot in the coming months, so if we had both `Grapher` and `PerSheetPreview`, we'd have to worry about and modify both whenever there's a change.

For now, how about we just have a simple stuntsheet selector and add/remove functionality?